src/database/queries/requestSessionMapping.js

```diff
@@ -0,0 +1,31 @@
+const requestSessionMapping = require('@database/models/index').SessionRequestMapping
+
+exports.addSessionRequest = async (requesteeId, requestId) => {
+	try {
+		const SessionRequestMappingData = [
+			{
+				requestee_id: requesteeId,
+				session_request_id: requestId,
```

Comment:
@sumanvpacewisdom change this session_request_id to request_session_id in all the places.